plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Indicates whether the module created the destination.
+    returned: success and if dest was created by the module.
+    type: bool
+    sample: true
+destination_attributes:
+    description: Attributes of a dest create by the module.
+    returned: success and destination was created by the module.
+    type: dict
+    contains:
+      block_size:
+        description:
+          Block size of the dataset.
+        type: int
+        sample: 32760
+      record_format:
+        description:
+          Content data type encoding.
```

Comment:
Here I'm not sure that the record format of a dataset is the encoding for the content it has.

https://www.ibm.com/docs/en/zos-basic-skills?topic=set-data-record-formats